<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>190</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UMich Class Watcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="Url_Edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>380</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>380</width>
      <height>10</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="Url_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LSA Course Guide URL</string>
    </property>
   </widget>
   <widget class="QLabel" name="Class_Watch_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Class #'s to watch:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Connection_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Load a URL</string>
    </property>
   </widget>
   <widget class="QLabel" name="Countdown_Label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClassEdit_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClassEdit_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClassEdit_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClassEdit_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClassEdit_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClassEdit_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start watching page.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logWindow">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Window</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="lWindow">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>380</width>
      <height>155</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ClassStat_1">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ClassStat_2">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ClassStat_3">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ClassStat_4">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>100</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ClassStat_5">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>100</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ClassStat_6">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>100</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="intervalInput">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Check Interval</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <zorder>Url_Edit</zorder>
   <zorder>progressBar</zorder>
   <zorder>Url_Label</zorder>
   <zorder>Class_Watch_label</zorder>
   <zorder>Connection_Label</zorder>
   <zorder>Countdown_Label</zorder>
   <zorder>ClassEdit_1</zorder>
   <zorder>ClassEdit_2</zorder>
   <zorder>ClassEdit_3</zorder>
   <zorder>ClassEdit_4</zorder>
   <zorder>ClassEdit_5</zorder>
   <zorder>ClassEdit_6</zorder>
   <zorder>StartButton</zorder>
   <zorder>logWindow</zorder>
   <zorder>ClassStat_1</zorder>
   <zorder>ClassStat_2</zorder>
   <zorder>ClassStat_3</zorder>
   <zorder>ClassStat_4</zorder>
   <zorder>ClassStat_5</zorder>
   <zorder>ClassStat_6</zorder>
   <zorder>intervalInput</zorder>
   <zorder>stopButton</zorder>
   <zorder>lWindow</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
